vets-service/src/main/java/com/petclinic/vets/datalayer/Vet.java

```diff
@@ -70,6 +70,9 @@
     @NotEmpty
     private String phoneNumber;
 
+    @Column(name = "profile_picture")
+    private String profilePicture;
+
```

Comment:
Isn't this supposed to be handled by another story and not yours?

Reply:
Changes have now been pushed.